### PR TITLE
 refactor posts context

### PR DESCRIPTION
**Pull request by hsnice16**
files to review

[postsSlice](https://github.com/hsnice16/Think-Loud/pull/19/files#diff-ef1904d61619bce3c42cdc4fff1300e41b8f4fabfcb2a1a340f59b71ddfc0f72)
[postsAPI](https://github.com/hsnice16/Think-Loud/pull/19/files#diff-1973864dae88cbe19e079ef11127730717f4ba64f2a3056cef868d385d69b0d5)

### CODE REVIEW

File: src/redux/features/posts/postsSlice.js [hunk 1]
```diff
@@ -23,7 +45,111 @@ export const postsSlice = createSlice({
...
+      .addMatcher(
+        postsAPI.endpoints.postEditedBroadcastCall.matchPending,
+        (state) => {
+          state.editBroadcastStatus = "loading";
+        }
+      )
+      .addMatcher(
+        postsAPI.endpoints.postEditedBroadcastCall.matchFulfilled,
+        (state, action) => {
+          state.editBroadcastStatus = "success";
```
Comment (vrx29) [inline src/redux/features/posts/postsSlice.js:124]: You can use constants here in place of success

**Review by vrx29 — APPROVED** [review]
Looks like just a refactoring of code. everything looks good. You can merge it.

File: src/redux/features/posts/postsSlice.js (hunk 1 above)
Comment (hsnice16) [reply src/redux/features/posts/postsSlice.js:124]: Thanks for suggesting, Will add in next PR.

**Review by Hamza-777 — APPROVED** [review]
Looks good.

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**